docs/SETUP.md

```diff
@@ -0,0 +1,107 @@
+# Setup
+
+Make a copy of _.env\_sample_ called _.env_ in the root of the project directory.
```

Comment:
Wrap in ticks... Think I made this comment elsewhere too? Not italics but `code`